### DISCUSSION TITLE
Amazon Used/In Stock Separation

### DISCUSSION THREAD

**Discussion by LiamGBR** [jef/streetmerchant#828] [Ideas and feature requests]
Looking for the possibility to only have brand new amazon items count as being "In Stock". One "Used-Like New" 5950x was in stock on amazon UK and caused notifications so wanted to ask if there is a way to ensure no pre owned or used items set off the in stock notifications.

Also wondering what terminal is being used at the bottom of StreetMerchant github

**Comment by thirdeyesteve** [comment]
I also agree with this addition. My notifications sound off for such items and I am only looking for new In stock,
